api/src/main/java/org/pmiops/workbench/profile/ProfileService.java

```diff
@@ -337,4 +391,303 @@ private void validateStringLength(String field, String fieldName, int max, int m
   public List<Profile> listAllProfiles() {
     return userService.getAllUsers().stream().map(this::getProfile).collect(Collectors.toList());
   }
+
+  private void verifyCaptcha(String captchaToken) {
```

Comment:
everything here is moved as-is from ProfileController except where noted.  Please do NOT review this moved code except to note whether it has been moved correctly.